torch/csrc/autograd/python_anomaly_mode.cpp

```diff
@@ -3,14 +3,70 @@
 #include <torch/csrc/Exceptions.h>
 #include <torch/csrc/autograd/python_anomaly_mode.h>
 #include <torch/csrc/autograd/python_cpp_function.h>
+#include <torch/csrc/profiler/combined_traceback.h>
 #include <torch/csrc/python_headers.h>
 #include <torch/csrc/utils/object_ptr.h>
 #include <torch/csrc/utils/pybind.h>
 #include <torch/csrc/utils/python_strings.h>
 
+#include <sstream>
+
 namespace torch::autograd {
 
+namespace {
+// Filter function matching the logic used by MemoryViz.js frameFilter
+bool shouldOmitFrame(const std::string& funcname, const std::string& filename) {
```

Comment:
Looking into potentially deduplicating this with MemoryViz, but doesn't look that easy. So left it like this for now.